src/middlewared/middlewared/plugins/pool.py

```diff
@@ -3481,8 +3493,9 @@ async def __common_validation(self, verrors, schema, data, mode, parent=None, cu
                 if data.get('acltype', 'INHERIT') == 'INHERIT':
                     to_check['acltype'] = check_ds['acltype']['value']
 
-                if to_check.get('acltype', 'POSIX') in ['POSIX', 'OFF'] and to_check.get('aclmode', 'DISCARD') != 'DISCARD':
-                    verrors.add(f'{schema}.aclmode', 'Must be set to DISCARD when acltype is POSIX or OFF')
+                if to_check.get('acltype', 'POSIX') in ['POSIX', 'OFF']:
+                    if to_check.get('aclmode', 'DISCARD') != 'DISCARD':
```

Comment:
Why 2 conditionals ;) ? We could just use brackets for using multiple lines or line breaks ?

Reply:
Because of personal preference. Line brackets or multiple lines makes the code messy, imo.